main.go

```diff
@@ -0,0 +1,170 @@
+// package main
```

Comment:
Please create directory and give it a name that shows what the function is doing.